src/main/java/seedu/address/model/person/Birthday.java

```diff
@@ -30,7 +34,25 @@ public Birthday(String birthday) {
      * Returns true if a given string is a valid birthday.
      */
     public static boolean isValidBirthday(String test) {
-        return test.matches(VALIDATION_REGEX);
+        boolean formatIsCorrect = test.matches(VALIDATION_REGEX);
```

Comment:
Is the checking of the regex still necessary if the block below is already checking if the date entered can be parsed?